site/book/01-getting-started/02-quickstart.md

```diff
@@ -44,21 +44,34 @@ $ vim deployment.yaml
 Often, you want to automatically mutate and/or validate resources in a package.
 `kpt fn` commands enable you to execute programs called _kpt functions_.
 
-For example, you can automatically set a label with key `env` on all the resources in the package:
+For example, you can automatically search and replace all the occurrences of `app` name on resources
+in the package using [path expressions]:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 -- env=dev
+$ kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.1 -- 'by-path=spec.**.app' 'put-value=my-nginx'
```

Comment:
getting this working is a blocker for fishfood now

Reply:
This does work now like in the demo. Did you face any problem getting this work.